Source/WebKit/UIProcess/API/glib/WebKitImage.cpp

```diff
@@ -247,21 +247,20 @@ guint webkit_image_get_stride(WebKitImage* image)
 }
 
 /**
- * webkit_image_as_rgba_bytes:
+ * webkit_image_as_bytes:
  * @image: a #WebKitImage
  *
  * Get the @image pixel data as an array of bytes.
  *
  * The pixel format for the returned byte buffer is 32-bit per pixel
  * with 8-bit premultiplied alpha, in the preferred byte order for
- * the architecture (typically ABGR8888 on little-endian hosts, and
- * RGBA8888 on big-endian ones).
+ * the architecture.
```

Comment:
I would mention here that the pixel format is specifically BGRA8888 because, after all, that's the hardcoded format in `WebImage::create`.